applications/crossbar/src/modules/cb_registrations.erl

```diff
@@ -198,36 +199,24 @@ merge_response(JObj, Regs) ->
                         end
                 end, Regs, kz_json:get_value(<<"Fields">>, JObj, [])).
 
+-spec maybe_default_port(integer(), nklib:scheme(), dict:dict()) -> integer().
+maybe_default_port(0, 'sips', _) -> 5061;
+maybe_default_port(0, 'sip', <<"TLS">>) -> 5061;
+maybe_default_port(0, 'sip', <<"tls">>) -> 5061;
+maybe_default_port(0, 'sip', _) -> 5060;
```

Comment:
`<<"TLS">>` or `<<"tls">>`  is correct.
to be more dialyzer friendly you can rewrite `Transport = props:get_value` to `Transport = props:get_binary_value`